manifests/config.pp

```diff
@@ -82,20 +82,16 @@
     require => File[$properties_dir]
   }
 
-  file { "${properties_dir}/admin.aclpolicy":
-    owner   => $user,
-    group   => $group,
-    mode    => '0640',
-    content => template($rundeck::acl_template),
-    require => File[$properties_dir]
+  rundeck::config::aclpolicyfile { 'admin':
+    acl_policies => $acl_policies,
+    owner        => $user,
```

Comment:
if the default owner and group are `rundeck::user` and `rundeck::group` we shouldn't need to specify them here.